src/page_body/structure/blocks/page_block_figma_component_props_table.pr

```diff
@@ -0,0 +1,92 @@
+{[ const block = context /]}
+
+{[ let configuration = exportConfiguration() /]}
+
+
+<div class="content-block content-block--figma-component-props-table" id="{{ getSearchIDString(block.id) }}">
+    {[ const component = ds.designComponentById(block.selectedFigmaComponent.figmaComponentId) /]}
```

Comment:
You should do this before rendering div: `<div class="content-block content-block--figma-component-props-tabl...` so in case component does not exist you won't render anything. So also wrap the whole table to the if and render table just in case component was found.